### DISCUSSION TITLE
Docker image releases

### DISCUSSION THREAD

**Discussion by upgreydd** [Budibase/budibase#5283] [General]
Hey. 
Have a question when updated docker images are released? 
We have new version v1.0.108 released but docker images are still v1.0.103.

**Comment by shogunpurple** [comment]
@Upgreydd We do monthly releases for big features, but will push out smaller stuff more regularly.

> We have new version v1.0.108 released but docker images are still v1.0.103.

I don't know what you are referring to here - v.1.0.108 is on dockerhub and it's been pushed. Updating with any of our update methods will upgrade you to the latest version:

- Clicking "update" in the update panel on a self hosted instance or upgrading with budibase 
- Upgrading manually using the `docker-compose` CLI
- Using the budibase CLI 

<img width="605" alt="Screenshot 2022-04-07 at 08 46 47" src="https://user-images.githubusercontent.com/11256663/162147502-af01913f-2697-40e5-832b-5c390fd9910d.png">

If your installation isn't updating, your `watchtower_service` might be down - you can verify this with a `docker ps` and bring it back up with `docker-compose up -d`

**Reply by upgreydd** [reply]
@shogunpurple OK, I've found an issue - problem is related to my app_service custom build - I'm attaching libraries there and `docker-compose up -d --build` is not downloading related source images.
I need to use:
```
docker-compose build --pull
docker-compose up -d
```

**Reply by oswaldB** [reply]
Hello, 

I am still locked in the 1.0.87 version and no possibility to upgrade.

the worse is that clicking on the upgrade button breaks my live apps...
{"message":"Database does not exist.","status":404}
